cartoframes/data/enrichment/points_enrichment.py

```diff
@@ -1,38 +1,58 @@
 import uuid
 
-from . import bigquery_client
+from ..clients import bigquery_client
 from ...auth import get_default_credentials
 from collections import defaultdict
 
 
 _ENRICHMENT_ID = 'enrichment_id'
+_WORKING_PROJECT = 'cartodb-on-gcp-datascience'
+_TEMP_DATASET_ENRICHMENT = 'enrichment_temp'
 
 # TODO: process column name in metadata, remove spaces and points
 
 
-def enrich_points(data, variables, data_geom_column='geometry', filters=dict()):
-
-    credentials = get_default_credentials()
+def enrich_points(data, variables, data_geom_column='geometry', filters=dict(), credentials=None):
 
+    credentials = credentials or get_default_credentials()
     bq_client = bigquery_client.BigQueryClient(credentials)
 
-    # Copy dataframe and generate id to join to original data later
-    data_copy = data.copy()
-    data_copy[_ENRICHMENT_ID] = range(data_copy.shape[0])
-    data_geometry_id_copy = data_copy[[_ENRICHMENT_ID, data_geom_column]]
+    data_copy = __copy_data_and_generate_enrichment_id(data, _ENRICHMENT_ID)
+
+    # Select only geometry and id and build schema
+    data_geometry_id_copy = data_copy[[data_geom_column, _ENRICHMENT_ID]]
+    schema = {data_geom_column: 'GEOGRAPHY', _ENRICHMENT_ID: 'INTEGER'}
 
+    # Data table is a universal unique identifier
     data_tablename = uuid.uuid4().hex
```

Comment:
Let's add a prefix or something to the table name to easily identify them, in case we have to do a bulk deletion of those tables.